src/base/bittorrent/session.cpp

```diff
@@ -91,7 +91,7 @@
 
 static const char PEER_ID[] = "qB";
 static const char RESUME_FOLDER[] = "BT_backup";
-static const char USER_AGENT[] = "qBittorrent " VERSION;
+static const char USER_AGENT[] = "qBittorrent/" QBT_VERSION_2;
```

Comment:
>We are also changing the whitespace to forward slash, I would guess they (tracker operators) need to add new rules anyway.

Shit I didn't realize that. I just meant "leave it the old way". But:

>Regardless of the change, we should mention this on www.qbittorrent.org website beforehand to minimize the impact.

What do you suggest? (assuming we change user-agent). Merge in master, and mention change in the news section after the release containing it?

Honestly, I don't download from trackers that whitelist clients so I have limited experience on this. I don't know which is more correct action. Leave it the old way and not annoy(wait) or tracker operators or just change and let the users deal with asshole operators?